src/elements/hx-checkbox-set/index.spec.js

```diff
@@ -0,0 +1,78 @@
+import { fixture, oneEvent, expect } from '@open-wc/testing';
+
+/**
+ * <hx-checkbox-set> component tests
+ *
+ * @type HXCheckboxSetElement
+ *
+ */
+describe('<hx-checkbox-set> component tests', () => {
```

Comment:
Please add tests for the following public methods:

* `isDirty`
* `wasChanged`
* `wasTouched`

https://github.com/HelixDesignSystem/helix-ui/blob/6237afdcefd9bb5bed01a7ee50555c5a0127a1e4/src/elements/hx-checkbox-set/index.js#L28-L51

Reply:
Changes have been done, Please review